liota/lib/transports/web_socket.py

```diff
@@ -76,60 +76,52 @@ def WebSocketConnection(self, host, verify_cert=True, CERTPATH="/etc/liota/cert"
             if self.ws is None:
                 raise (IOError("Couldn't verify host certificate"))
 
-    def run(self):
+    def receive(self, queue):
```

Comment:
This will be fixed when we move to WebSocketApp implementation marked as an issue.